resources/legacy_template.tex

```diff
@@ -0,0 +1,30 @@
+% This template avoids the scontents package, which is only available on relatively recent TeX distributions
+\documentclass[varwidth=true, crop, border=5pt]{standalone}
+
+% Packages
+\usepackage{amsmath}
+\usepackage{amssymb}
+
+% Blank formula checking
+\usepackage{ifthen}
+\newlength{\pheight}
+
+% Color support
+\usepackage{xcolor}
+\definecolor{xpp_font_color}{HTML}{%%XPP_TEXT_COLOR%%}
+
+% User input
+\def\preview{\( 
+    \displaystyle
+    %%XPP_TOOL_INPUT%%
+    \)
+}
```

Comment:
This will add a redundant space at the end of TeX object.  `%` should be added after `\)`.

Reply:
Thanks for the catch. I amended it to the previous commit.